scripts/package-update.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Unix shell script for updating all packages in the project
+
+# Update root dependencies
+bun update
+
+# Update apps
+cd apps
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)